data/Person.csv

```diff
@@ -63,16 +63,16 @@ P0031,Huang,Guangzu,en,male,no,NULL,NULL,NULL,1946,,,,PL0012,S03,yes,T00109,ONLI
 P0031,黄,光祖,zh,male,no,NULL,NULL,NULL,1946,,,,PL0012,S03,yes,T00109,ONLINE SOURCE,http://irrc.swu.edu.cn/s/irrc/gw/20160415/902381.html,ONLINE SOURCE,川东师范学堂第二十三任校长黄子裳之子 。 Aus Remembrance: [朱]时贤读的东西比较多，在部队中接触过一些内部资料，如《我军不利战战史》，这让他对众多问题的看法发生 了重大变化。,,2017-07-05,LH,2018-01-18,OS
 P0032,Zhu,Shixian,en,male,unknown,NULL,NULL,PL0006,1944,1972,,,,,yes,T00109,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-25,WH
 P0032,朱,时贤,zh,male,unknown,NULL,NULL,PL0006,1944,1972,,,,,yes,T00109,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-25,WH
-P0033,Zhou,Ziren,en,male,unknown,NULL,NULL,PL0007,1943?,,,,,,,T00116,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-26,DP
-P0033,周,孜仁,zh,male,unknown,NULL,NULL,PL0007,1943?,,,,,,,T00116,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-26,DP
+P0033,Zhou,Ziren,en,male,,,,PL0007,1943?,,,,PL0004,S06,no,T00116,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-25,OS
+P0033,周,孜仁,zh,male,,,,PL0007,1943?,,,,PL0004,S06,no,T00116,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-25,OS
 P0034,Zhou,Jiren,en,male,no,NULL,NULL,PL0006,,,Zhou Zhaoming,en,,,,T00116,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-25,WH
 P0034,周,继仁,zh,male,no,NULL,NULL,PL0006,,,周照明,zh,,,,T00116,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-25,WH
 P0035,Zhang,Kunquan,en,female,NULL,NULL,NULL,,1926,,Shamo,en,PL0100,,,T00117,ONLINE SOURCE,,,,,2017-07-05,LH,2018-02-06,WH
 P0035,张,坤权,zh,female,NULL,NULL,NULL,,1926,,沙漠,zh,PL0100,,,T00117,ONLINE SOURCE,,,,,2017-07-05,LH,2018-02-06,WH
 P0036,Huang,Zhongjing,en,male,unknown,NULL,NULL,,,,,,,,,T00118,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-25,WH
 P0036,黄,中敬,zh,male,unknown,NULL,NULL,,,,,,,,,T00118,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-25,WH
-P0037,Huang,Xiaozhen,en,male,yes,1964,1972,PL0009,1947,1995,,,,,,T00118,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-25,WH
-P0037,黄,小振,zh,male,yes,1964,1972,PL0009,1947,1995,,,,,,T00118,ONLINE SOURCE,,,,,2017-07-05,LH,2018-01-25,WH
+P0037,Huang,Xiaozhen,en,male,,1965,1973,PL0009,1947,1995,,,,,,T00118,ONLINE SOURCE,,,,,2017-07-05,LH,2018-02-06,OS
```

Comment:
this seems to be the wrong change, the yes is in line with the dates provided later, not sure how to settle on the correct birth and death dates

Reply:
Here is a "yes" missing. But I don't think that something is wrong with the date (both rusticated and birth). Can you tell me again what you mean?

Reply:
in @inamoto4 version ismissing `yes` compared to yours, but the birth and death date for Mr Huang are also different. I'm not sure which it should be.

Reply:
I looked up on the internet, the birth and death date seem to be 1947 and 1996. I changed them and added a yes already. :)

Reply:
you are right I meant the rustication dates 1964 vs 1965, and 1972 vs 1973

Reply:
There stands from T00118 only that Huang was 17 years old as he went in to the countryside and he spends 8 years there. It could be 1964-1972 or 1965-1973. I found a new source which shows the correct time:  
https://www.douban.com/event/16083771/

Reply:
Yes, I have used that version as well. and therefore changed it. But I guess I did not change the death date. Maybe @duncdrum can commit this information directly into the Master branch, including the second source.